components/YesterdayWidget.tsx

```
@@ -0,0 +1,121 @@
+        ) : (
+          <p className="mt-2 text-sm text-muted">
+            Summary bullets will appear after the next run.
+          </p>
+        )}
+      </div>
+
+      {isExpanded && (
+        <div className="mt-6 border-t border-border/70 pt-6">
+          <div
+            className="prose-luxury max-w-none"
+            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)